lib/libutee/base64.c

```diff
@@ -84,7 +84,7 @@ bool base64_dec(const char *data, size_t size, void *buf, size_t *blen)
 		if (!get_idx(data[n], &idx))
 			continue;
 
-		if (m >= *blen)
+		if (m > *blen)
```

Comment:
This doesn't look good, because buffer is valid from offest 0 to `*blen - 1` so writing at offset `*blen` should not be allowed.

Reply:
you're right (again :), thanks for the review).
I'll check back were the original issue reported by #3876 is.